model/articles.model.js

```diff
@@ -38,3 +38,42 @@ exports.checkArticleExist = (article_id)=>{
     })
 }
 
+exports.updateVote = (article_id, inc_votes) => {
+    if (inc_votes === 0 || typeof inc_votes !== 'number') {
+      return Promise.reject({ status: 400, msg: 'Bad request' });
+    }
+  
+    return db.query('SELECT votes FROM articles WHERE article_id = $1', [article_id])
+      .then((result) => {
+        if (result.rows.length === 0) {
+          return Promise.reject({ status: 404, msg: 'Article not found' });
+        }
+  
+        const currentVotes = result.rows[0].votes;
+        const updatedVotes = currentVotes + inc_votes;
+  
+        if (updatedVotes < 0) {
+          return Promise.reject({ status: 400, msg: 'Bad request' });
+        }
+  
+        let updateQuery;
+        let queryParams;
+  
+        if (inc_votes > 0) {
```

Comment:
if you try to add a minus, you get a minus - so this potentially isnt needed, but up to you. Did it fail the test without it?